index.html

```diff
@@ -163,9 +164,15 @@ <h1 id="title"><i class="todo">[DRAFT]</i> Media Working Group Charter</h1>
         </table>
       </div>
 
+      <div id="background" class="background">
+        <h2>Motivation and Background</h2>
+        <p>Standardization efforts to develop media foundations for the Web, such as the <a href="https://html.spec.whatwg.org/#htmlmediaelement"><code>HTMLMediaElement</code></a> interface and <a href="https://www.w3.org/TR/media-source/">Media Source Extensions</a>, have helped turn the Web into a major platform for media streaming and media consumption.</p>
+        <p>Experience gained through implementation, deployment and usage of these technologies, standardization of real-time technologies within the <a href="https://www.w3.org/groups/wg/webrtc/">Web Real-Time Communications Working Group</a>, incubation discussions within the <a href="https://wicg.io/">Web Platform Incubator Community Group</a>, and general feedback from the media industry, create new requirements and opportunities to improve the overall media playback experience on the web.</p>
```

Comment:
As above, our scope is a bit broader than media playback experience.

Reply:
I reformulated and also adjusted the first sentence in the scope section that somewhat incorrectly focused on "playback" as well. Or rather, I dropped that sentence and moved it to the background section since it's more aspirational than truly meant at setting the scope.